woocommerce/woocommerce-ios
# Bundle products in order form: storage layer changes

Part of #10428 


## Why


To support a bundle product's configuration on each bundled item which can be a variable product with allowed variations and default variations, a few attributes and a relationship are required for the `ProductBundleItem` entity.

## How

For an easier review of the Core Data model diffs, please check out the commits after the first one. I went through some changes in a few commits while working on a barebone flow based on these changes, and I believe these are the minimum required attributes/relationships to support the configuration UX similar to core.

## Testing instructions


A confidence check to run the app on another branch in the logged-in state, then run the PR branch to make sure everything works as before would be great.


---
